kayenta-signalfx/src/integration-test/resources/integration-test-canary-config.json

```diff
@@ -26,18 +26,7 @@
     {
       "name": "Bad Request Rate for /v1/some-endpoint",
       "query": {
-        "metricName": "kayenta.integration-test.request.count",
-        "queryPairs": [
-          {
-            "key": "uri",
-            "value": "/v1/some-endpoint"
-          },
-          {
-            "key": "status_code",
-            "value": "4*"
-          }
-        ],
-        "aggregationMethod": "sum",
+        "customInlineTemplate": "data('kayenta.integration-test.request.count', filter=filter('canary-scope', '${scope}') and filter('location', '${location}')).mean(by=['location', 'canary-scope']).publish()",
```

Comment:
It's not a huge deal but can we break this up at all, it's kind of difficult to parse given how long the line is.

Reply:
The content of this `customInlineTemplate` looks different from what got replaced e.g. the `status_code` is missing. Any reason to this?

Reply:
Oh good point, I copied the content from the `customInlineTemplate` above and missed adding the additional filters